apply/golden_file_test.go

```diff
@@ -60,18 +61,26 @@ func TestIntegration(t *testing.T) {
 				e = apply.Apply(testdataFs, conf, templates.Templates, true)
 				a.NoError(e)
 			} else {
-
+				fileName := "fogg.yml"
 				fs, _, e := util.TestFs()
 				a.NoError(e)
 
 				// copy fogg.json into the tmp test dir (so that it doesn't show up as a diff)
-				configContents, e := afero.ReadFile(testdataFs, "fogg.json")
+				configContents, e := afero.ReadFile(testdataFs, "fogg.yml")
+				if os.IsNotExist(e) { //If the error is related to the file being non-existent
+					configContents, e = afero.ReadFile(testdataFs, "fogg.json")
+					fileName = "fogg.json"
+				}
 				a.NoError(e)
-				configMode, e := testdataFs.Stat("fogg.json")
+
+				var configMode os.FileInfo
```

Comment:
you can now get rid of these `var` declarations and do the inline assignments on L79,83